packages/react-instantsearch-core/src/widgets/Index.tsx

```diff
@@ -12,6 +12,7 @@ function getIndexContext(props: Props): IndexContext {
 type Props = {
   indexName: string;
   indexId: string;
+  children?: React.ReactNode;
 };
```

Comment:
An alternative would be:

```suggestion
type Props = React.PropsWithChildren<{
  indexName: string;
  indexId: string;
}>;
```

I don't know if one way is better than the other, though, so feel free to disregard if you're satisfied with the current type definition.